File: C# Advanced/C# Advanced - Exercises/Exercise-03-Sets and Dictionaries/Task03.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace Practice
{
    class Program
    {
        static void Main(string[] args)
        {
            int n = int.Parse(Console.ReadLine());

            SortedSet<string> set = new SortedSet<string>();

            for (int i = 0; i < n; i++)
            {
                string[] input = Console.ReadLine().Split();

                foreach (var item in input)
                {
                    set.Add(item);
                }
            }

            Console.WriteLine(String.Join(" ", set));

        }
    }
}

File: JS Fundamentals/JS Fundamentals - Labs/Lab-05-Arrays Advanced/Task03.js
function solve(input) {
    let k = input.shift();

    console.log(input.slice(0, k).join(' '));
    console.log(input.slice(input.length - k, input.length).join(' '));
}

solve([2, 7, 8, 9, 10, 20, 30]);